Elevel/Persistence/Context/ApplicationDbContextSeed.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Threading.Tasks;
+using Elevel.Domain;
+using Elevel.Domain.Models;
+using Microsoft.AspNetCore.Identity;
+
+namespace Elevel.Infrastructure.Persistence.Context
+{
+    public class ApplicationDbContextSeed
+    {
+        public static async Task SeedEssentialsAsync(UserManager<ApplicationUser> userManager,
+            RoleManager<IdentityRole> roleManager)
+        {
+            //Seed Roles
+            await roleManager.CreateAsync(new IdentityRole(Authorization.Roles.Administrator.ToString()));
```

Comment:
before you should check, that DB is not contain anything, in other case each run you will add copy

Reply:
I checked - no duplicates are added. I think that UserManager and RoleManager already have built-in check for adding duplicates.